eZ/Publish/API/Repository/NotificationService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
```

Comment:
Please add `declare(strict_types=1);` and proper type hints for parameters and return values.